<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>achive</class>
 <widget class="QWidget" name="achive">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="achiveLabel_5">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>282</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="achiveLabel_9">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>430</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="achiveLabel_3">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>132</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="achiveLabel_6">
   <property name="geometry">
    <rect>
     <x>269</x>
     <y>282</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="achiveLabel_4">
   <property name="geometry">
    <rect>
     <x>28</x>
     <y>282</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="achiveLabel_2">
   <property name="geometry">
    <rect>
     <x>269</x>
     <y>132</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="achiveLabel_1">
   <property name="geometry">
    <rect>
     <x>28</x>
     <y>132</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="achiveLabel_7">
   <property name="geometry">
    <rect>
     <x>28</x>
     <y>430</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="achiveLabel_8">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>430</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
